packages/pieces/community/mailer-lite/package.json

```diff
@@ -1,4 +1,4 @@
 {
   "name": "@activepieces/piece-mailer-lite",
-  "version": "0.4.1"
+  "version": "1.0.1"
```

Comment:
You can increment the middle version If it's less than 1 and It won't be automatically upgraded.

I am changing this version to 0.5.0 

https://www.activepieces.com/docs/developers/piece-reference/piece-versioning